src/WebApiDebugger/Program.cs

```diff
@@ -9,6 +9,8 @@
 app.UseSwagger();
 app.UseSwaggerUI();
 
+throw new Exception("Some exception throw while configuring the application");
```

Comment:
I don't understand why we throw here?

Reply:
I think that this is meant to be an example project that will be debugged when running `msbuild /t:ValidateOpenApi` to debug the actual msbuild task according to new information in the `README`. Correct me if I'm wrong 😹

Reply:
I believe this was used during development for debugging purposes. This line of code makes this launch profile  fail. Basically we can't extract the OpenAPI spec from the code because the Swagger CLI tool runs this piece of code, which fails...

![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/14242083/bfb2da9a-b7f0-431f-bc60-e77d928a4883)

Reply:
yeah 100% use to debug the behavvior when throwing the exception XD